static/app/utils/replays/frame.tsx

```diff
@@ -25,11 +26,9 @@ export function getColor(frame: ReplayFrame): Color {
       case 'issue':
         return 'red300';
       case 'ui.slowClickDetected':
-        return (frame as SlowClickFrame).data.endReason === 'timeout'
-          ? 'red300'
-          : 'yellow300';
+        return isDeadClick(frame as SlowClickFrame) ? 'red300' : 'yellow300';
       case 'ui.multiClick':
-        return 'red300';
+        return isRageClick(frame as MultiClickFrame) ? 'red300' : 'yellow300';
```

Comment:
rage clicks should be coming from `slowClickDetected`:

https://github.com/getsentry/sentry/pull/53140/files#diff-35d4d3e9bf8968daac8b8ed2cb759eced2ab24c5a51d26fa1e2e3e525571e85eR142-R150